lib/tasks/migrations.rake

```diff
@@ -19,9 +25,12 @@ namespace :migrations do
     Dir.foreach(dir_name){|file_name| puts file_name;
       if file_name != '.' && file_name != '..';
         begin
-          key = Aws::S3::Key.create(bucket, 'uploads/images/' + file_name);
-          key.put(File.open(dir_name+ '/' + file_name).read, 'public-read');
-          key.public_link();
+          file = bucket.files.new({
+            key:    "uploads/images/#{file_name}",
```

Comment:
Layout/FirstHashElementIndentation: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.